src/rc/start-stop-daemon.c

```diff
@@ -549,6 +560,14 @@ int main(int argc, char **argv)
 			stderr_process = optarg;
 			break;
 
+		case 0x81: /* --scheduler "Linux Process scheduler index" */
+#ifdef HAVE_SCHED
+			sscanf(optarg, "%d:%d", &scheduler, &sched_prio);
```

Comment:
this makes the underlying constants part of the CLI which doesn't work as those values are not defined by POSIX.  i think you need to have the CLI specify symbolic names (e.g. `fifo`), and then manually convert that to `SCHED_FIFO` and such.